ws-nextjs-app/pages/[service]/live/[id]/Post/index.tsx

```diff
@@ -144,6 +144,7 @@ const PostContent = ({ contentBlocks }: { contentBlocks: OptimoBlock[] }) => {
         sizes="(min-width: 1008px) 760px, 100vw"
         className="mediaStyles"
         css={styles.bodyMedia}
+        position={[9]}
```

Comment:
Is this just to ensure it's not preloaded?

Reply:
yes - we just want all images to lazy load. The only one we don't want to lazy load is the header image. I could make it clearer somehow?

Reply:
The API is a little weird for that component, so if it works maybe just leave it as you have here and we can come back to it. 

A bit hard to reason about within the component as it tries to automatically determine preloading and lazyloading when I feel like they should be done at the prop level.